generators/typescript/sdk/versions.yml

```diff
@@ -0,0 +1,1217 @@
+- version: 1.0.0
+  changelogEntry:
+    - summary: Change default configuration values for several options
+      type: feat
+    - summary: Fix property name quoting in interfaces and classes to only use quotes when necessary
+      type: fix
+  createdAt: '2025-05-14'
+  irVersion: 57
```

Comment:
🚀

Reply:
whoa V1'ing?!

Reply:
That happened earlier this week :) This PR just moves to versions.yml